kots.yaml

```diff
@@ -21,26 +21,15 @@ pipeline:
   - uses: fetch
     with:
       uri: https://github.com/replicatedhq/kots/archive/refs/tags/v${{package.version}}.tar.gz
-      expected-sha256: cb431fc87d699d8f4ec805eb1480156a899aec091f4d5ebadfec41f45fb9f59f
+      expected-sha256: d2c0bb6717f759355c5d70bb37b8ba4b95dac0fc4ac0b20bd904ff47e1e2322a
 
   - runs: |
       set -x
-      export DESTDIR="${{targets.destdir}}"
-      mkdir -p "${DESTDIR}"
+      mkdir -p ${{targets.destdir}}/etc
+      mkdir -p ${{targets.destdir}}/usr/bin
 
       # Scripts etc.
-      mv deploy/assets/backup.sh "${DESTDIR}/backup.sh"
-      mv deploy/assets/restore-db.sh "${DESTDIR}/restore-db.sh"
-      mv deploy/assets/restore-s3.sh "${DESTDIR}/restore-s3.sh"
-      mv deploy/assets/restore.sh "${DESTDIR}/restore.sh"
-      mv deploy/assets/migrate-s3.sh "${DESTDIR}/migrate-s3.sh"
-      mv deploy/assets/fs-minio-check.sh "${DESTDIR}/fs-minio-check.sh"
-      mv deploy/assets/fs-minio-reset.sh "${DESTDIR}/fs-minio-reset.sh"
-      mv deploy/assets/fs-minio-keys-sha.sh "${DESTDIR}/fs-minio-keys-sha.sh"
-      mv deploy/assets/s3-bucket-create.sh "${DESTDIR}/s3-bucket-create.sh"
-      mv deploy/assets/s3-bucket-head.sh "${DESTDIR}/s3-bucket-head.sh"
-      mv deploy/assets/kots-upgrade.sh "${DESTDIR}/kots-upgrade.sh"
-      mv deploy/assets/postgres "${DESTDIR}/postgres"
+      mv deploy/assets/* "${{targets.destdir}}/etc/"
```

Comment:
Just curious if there was a reason these files were explicitly named before, like if there are additional files we don't want to be added to the APK?

Reply:
There's no extra files but does mean we wont get any new ones added upstream that come in with automated updates.  This should prevent issues with missing deploy assets in the future.

Reply:
Ah, very nice!!